inginious/frontend/plugins/manual_scoring/pages/manual_scoring.py

```diff
@@ -0,0 +1,124 @@
+        course, task = self.get_course_and_check_rights(course_id, task_id)
+
+        self.update_manual_comment_and_grade(submission_id)
+        self.add_css_and_js_file()
+        return self.render_page(course, task, submission_id)
+
+    def update_manual_comment_and_grade(self, submission_id):
+        """ update the grade and comment on db """
+        manual_grade = get_mandatory_parameter(web.input(), "manual_grade")
+        comment = get_mandatory_parameter(web.input(), "comment")
+        rubric_status = get_mandatory_parameter(web.input(), "rubric")
+        self.database.submissions.update(
+            {"_id": ObjectId(submission_id)},
+            {"$set": {"manual_scoring.grade": manual_grade,
+                      "manual_scoring.comment": comment,
+                      "manual_scoring.rubric_status": rubric_status
+                      }
+             })
+
+    def render_page(self, course, task, submission_id):
+        """ Get all data and display the page """
+        rubric_content = self.get_rubric_content()
+        problem_id = task.get_problems()[0].get_id()
+        submission = self.submission_manager.get_submission(submission_id, user_check=False)
+        submission_input = self.submission_manager.get_input_from_submission(submission)
+        comment, score, rubric_status = get_manual_scoring_data(submission)
+
+        data = {
+            "url": 'manual_scoring',
+            "summary": submission_input['custom']['custom_summary_result'],
+            "grade": score,
+            "language": submission_input['input'][problem_id + '/language'],
+            "comment": comment,
+            "score": submission_input['grade'],
+            "task_name": course.get_task(submission_input['taskid']).get_name(self.user_manager.session_language()),
+            "result": submission_input['result'],
+            "feedback_result_text": get_submission_result_text(submission_input),
+            "problem": submission_input['input'][problem_id],
+            "username": submission_input['username'][0],
+            "name": self.user_manager.get_user_realname(submission_input['username'][0]),
+            "env": task.get_environment(),
+            "question_id": problem_id,
+            "submission_id": submission_id,
+            "rubric_status": rubric_status
+        }
+
+        return (
+            self.template_helper.get_custom_renderer(base_renderer_path).manual_scoring(
+                course, task, rubric_content, data)
+        )
+
+    def get_rubric_content(self):
+        """ return the content of the rubric depending of the language """
+        path = 'inginious/frontend/plugins/manual_scoring/static/json/'
+        language_file = {'es': 'rubric_es.json', 'en': 'rubric.json', 'de': 'rubric.json', 'fr': 'rubric.json',
+                         'pt': 'rubric.json'}
+        current_language = self.user_manager.session_language()
+        path += language_file[current_language]
```

Comment:
import the `os` module and use it here to avoid other kinds of problems. I think it is as follows, but please check first.

```suggestion
        path = os.path.join(path, language_file[current_language])
```